adapters/Tapjoy/README.md

```diff
@@ -1,25 +1,44 @@
-Steps to generate universal static library and framework for iOS Devices and
-Simulators with supported architectures.
+# Tapjoy Mediation Adapter for Google Mobile Ads SDK for iOS
 
-Prerequisites:
+## Prerequisites
 - Xcode 6.0 or higher
-- Deployment target of 6.0 or higher
-- Google Mobile Ads SDK
-- Tapjoy SDK
-- Tapjoy Adapter Source Code
+- iOS Deployment target of 7.0 or higher
+- Minimum required Google Mobile Ads SDK 7.10.1
+- Minimum required Tapjoy SDK 11.9.1
 
-Setup Instructions:
-- Drop GoogleMobileAds framework to
-  Project Directory->Drop_Framework_And_Headers.
-- Drop Tapjoy framework to Project Directory->Drop_Framework_And_Headers.
+## Instructions
+- Add the Google Mobile Ads SDK. See the
+  [quick start guide](https://firebase.google.com/docs/admob/ios/quick-start)
+  for detailed instructions on how to integrate the Google Mobile Ads SDK.
+- Add or drag the adapter .framework into your Xcode project.
+- Drag the Tapjoy framework into your Xcode project. You can find the
+  Tapjoy SDK [here](http://dev.tapjoy.com/sdk-integration/ios).
+- Enable the Ad network in the Ad Network Mediation UI.
+- TapjoyAdapter framework has a `GADMTapjoyExtras` class to provide
+  `debugEnabled` parameter. The `debugEnabled` is used to enabled logs for
```

Comment:
It will be more appropriate to say "The `debugEnabled` can be used to enable logs........."